admin/README.md

```diff
@@ -0,0 +1,297 @@
+# Gaia Admin Service
+
+This is a small service you can co-locate with your Gaia hub that lets you
+remotely administer it with an API key.  Using this service, you can:
+
+* Add, change, and remove whitelisted writers
+* Set and change your backend storage driver
+* Set and change your Gaia hub's connection settings
+* Restart your Gaia hub to make the changes take effect
+
+# Usage
+
+## Building and Running
+
+You can build this service from this directory as follows:
+
+```
+$ npm run build
+```
+
+To run in place:
+
+```
+$ node lib/index.js [/path/to/config.json]
+```
+
+To install this service as the `blockstack-gaia-admin` program in your `$PATH`:
+
+```
+$ sudo npm install -g
+$ which blockstack-gaia-admin
+/usr/bin/blockstack-gaia-admin
+```
+
+## Configuration
+
+The admin service needs a separate config file from your Gaia hub.  Importantly,
+it needs to know the following:
+
+* where the Gaia hub config file is located
+* what API key(s) will be used to authenticate administrative requests
+* what command(s) to run to restart the Gaia hub on a config change
+
+Here is a sample config file for a Gaia hub config located at
+`/tmp/gaia-config.json`, and with a single API key "`hello`".  The reload
+command is set to kill all instances of the `blockstack-gaia-hub` program, and
+restart it with `nohup`.  You should tailor this to your deployment.
+
+```bash
+$ cat /tmp/gaia-admin.json
+{
+   "argsTransport": {
+    "level": "debug",
+    "handleExceptions": true,
+    "timestamp": true,
+    "stringify": true,
+    "colorize": true,
+    "json": true
+  },
+  "regtest": false,
+  "testnet": false,
```

Comment:
what do the testnet/regtest configuration options do?

Reply:
Nothing yet.